uber/templates/preregistration/preregbase.html

```diff
@@ -133,7 +133,7 @@
             }
         };
         var makeBadgeMatchExtra = function () {
-            if ($(BADGE_TYPES.row).size()) {
+            if ($(BADGE_TYPES.row).size() && BADGE_TYPES.options[0]) {
```

Comment:
A more idiomatic way to say this would be ``BADGE_TYPES.options.length`` or better yet ``_(BADGE_TYPES.options).size()``.